nipype/interfaces/freesurfer/longitudinal.py

```diff
@@ -251,3 +253,207 @@ def _list_outputs(self):
         outputs = self.output_spec().get()
         outputs["out_file"] = os.path.abspath(self.inputs.out_file)
         return outputs
+
+
+class BaseReconAllInputSpec(ReconAllInputSpec):
+    subject_id = traits.Str(argstr="-subjid %s", desc="subject name")
+    base_id = traits.Str(argstr="-base %s", desc="base template name", xor=["subject_id"])
+    timepoints = InputMultiObject(
+        traits.Str(), argstr="-tp %s...", desc="processed time point to use in template"
+    )
```

Comment:
A few suggestions here

Reply:
This proposal adds more explicitness to the argument names as to what they refer to. Both points to subject identifiers in FreeSurfer's subjects directory, hence the `id` suffix. We also try to favour long form of arguments, hence `tp` -> `-base-tp`.

```suggestion
    base_template_id = traits.Str(
        argstr="-base %s",
        desc="base template name",
        mandatory=True,
    )
    base_timepoint_ids = InputMultiObject(
        traits.Str(),
        argstr="-base-tp %s...",
        desc="processed time point to use in template",
        mandatory=True,
    )
```